src/proctored-exam-settings/ProctoredExamSettings.test.jsx

```diff
@@ -123,3 +124,95 @@ describe('ProctoredExamSettings connection states tests', () => {
     cleanup();
   });
 });
+
+describe('ProctoredExamSettings save settings tests', () => {
+  const mockGetData = {
+    data: {
+      proctored_exam_settings: {
+        enable_proctored_exams: true,
+        allow_proctoring_opt_out: false,
+        proctoring_provider: 'mockproc',
+        proctoring_escalation_email: 'test@example.com',
+        create_zendesk_tickets: true,
+      },
+      available_proctoring_providers: ['software_secure', 'proctortrack', 'mockproc'],
+    },
+  };
+
+  function mockAPI(getData, postResult) {
+    const mockClientGet = jest.fn(async () => (getData));
+    const mockClientPost = postResult ? jest.fn(async () => (postResult)) : jest.fn(async () => { throw new Error(); });
+    auth.getAuthenticatedHttpClient = jest.fn(() => ({
+      get: mockClientGet,
+      post: mockClientPost,
+    }));
+    auth.getAuthenticatedUser = jest.fn(() => ({ userId: 3 }));
+    return { mockClientGet, mockClientPost };
+  }
+
+  it('Makes API call successfully', async () => {
+    const mockedFunctions = mockAPI(mockGetData, { data: 'success' });
+    await act(async () => render(<ProctoredExamSettings {...defaultProps} />));
+    // Make a change to the provider to proctortrack and set the email
+    const selectElement = screen.getByDisplayValue('mockproc');
+    await act(async () => {
+      fireEvent.change(selectElement, { target: { value: 'proctortrack' } });
+    });
+    const escalationEmail = screen.getByTestId('escalationEmail');
+    expect(escalationEmail.value).toEqual('test@example.com');
+    await act(async () => {
+      fireEvent.change(escalationEmail, { target: { value: 'proctortrack@example.com' } });
+    });
+    expect(escalationEmail.value).toEqual('proctortrack@example.com');
+    const submitButton = screen.getByTestId('submissionButton');
+    await act(async () => {
+      fireEvent.click(submitButton, { target: { value: 'proctortrack@example.com' } });
```

Comment:
I'm a little confused here. Why is the event `{ target: { value: 'proctortrack@example.com' } }` for this button click? Is this copy paste from the above `fireEvent` call?